apps/emqx_rule_engine/src/emqx_rule_engine_jwt.erl

```diff
@@ -43,3 +44,13 @@ lookup_jwt(TId, ResourceId) ->
         error:badarg ->
             {error, not_found}
     end.
+
+-spec delete_jwt(ets:table(), resource_id()) -> ok.
+delete_jwt(TId, ResourceId) ->
+    try
+        ets:delete(TId, {ResourceId, jwt}),
```

Comment:
nit: is dialyzer happy with this?
```suggestion
        _ = ets:delete(TId, {ResourceId, jwt}),
```

Reply:
according to CI, it is.

probably because `ets:delete/2` always returns `true`, so there are not multiple values to match on.